ColorizeVolume/ColorizeVolume.py

```diff
@@ -119,24 +133,48 @@ def setup(self) -> None:
         # Connections
 
         # These connections ensure that we update parameter node when scene is closed
-        self.addObserver(slicer.mrmlScene, slicer.mrmlScene.StartCloseEvent, self.onSceneStartClose)
-        self.addObserver(slicer.mrmlScene, slicer.mrmlScene.EndCloseEvent, self.onSceneEndClose)
+        self.addObserver(
+            slicer.mrmlScene, slicer.mrmlScene.StartCloseEvent, self.onSceneStartClose
+        )
+        self.addObserver(
+            slicer.mrmlScene, slicer.mrmlScene.EndCloseEvent, self.onSceneEndClose
+        )
 
-        self.ui.outputRgbaVolumeSelector.connect('currentNodeChanged(vtkMRMLNode*)', self.onOutputRgbaVolumeSelected)
+        self.ui.outputRgbaVolumeSelector.connect(
+            "currentNodeChanged(vtkMRMLNode*)", self.onOutputRgbaVolumeSelected
+        )
 
         # Buttons
-        self.ui.applyButton.connect('clicked(bool)', self.onApplyButton)
-        self.ui.showVolumeRenderingButton.connect('clicked(bool)', self.onShowButton)
-        self.ui.resetVolumeRenderingSettingsButton.connect('clicked(bool)', self.onResetVolumeRenderingSettingsButton)
-        self.ui.volumeRenderingSettingsButton.connect('clicked(bool)', self.onVolumeRenderingSettingsButton)
-        self.ui.resetToDefaultsButton.connect('clicked(bool)', self.onResetToDefaultsButton)
-
-        self.ui.volumeRenderingLevelWidget.connect('valueChanged(double)', self.onUpdateVolumeRenderingTransferFunction)
-        self.ui.volumeRenderingWindowWidget.connect('valueChanged(double)', self.onUpdateVolumeRenderingTransferFunction)
-        self.ui.volumeRenderingOpacityWidget.connect('valueChanged(double)', self.onUpdateVolumeRenderingTransferFunction)
-        self.ui.volumeRenderingGradientOpacityCheckBox.connect('toggled(bool)', self.onUpdateVolumeRenderingTransferFunction)
-        self.ui.volumeRenderingGradientOpacityLevelWidget.connect('valueChanged(double)', self.onUpdateVolumeRenderingTransferFunction)
-        self.ui.volumeRenderingGradientOpacityWindowWidget.connect('valueChanged(double)', self.onUpdateVolumeRenderingTransferFunction)
+        self.ui.applyButton.connect("clicked(bool)", self.onApplyButton)
+        self.ui.showVolumeRenderingButton.connect("clicked(bool)", self.onShowButton)
+        self.ui.resetVolumeRenderingSettingsButton.connect(
+            "clicked(bool)", self.onResetVolumeRenderingSettingsButton
+        )
+        self.ui.volumeRenderingSettingsButton.connect(
+            "clicked(bool)", self.onVolumeRenderingSettingsButton
+        )
+        self.ui.resetToDefaultsButton.connect(
+            "clicked(bool)", self.onResetToDefaultsButton
+        )
+
+        self.ui.volumeRenderingLevelWidget.connect(
+            "valueChanged(double)", self.onUpdateVolumeRenderingTransferFunction
```

Comment:
OK, I stop commenting on this, just revert all these breakups of single-line commands into 3 lines. It just does not make any sense.